src/stores/viewStore.ts

```diff
@@ -109,13 +111,9 @@ const useViewStore = create<ViewState>((set, get) => ({
         const {setUiState} = useUiStore.getState();
         setUiState(UI_STATE.FAST_LOADING);
 
-        const {clearQuery} = useQueryStore.getState();
-        clearQuery();
```

Comment:
somehow this comment was missed in the last batch -

good catch! let's also remove the one in logFileStore.ts: loadFile(); otherwise, this link won't load the initial query: http://localhost:3010/?filePath=/test/test.clp.zst#logEventNum=999&queryString=hi